elide-graphql/src/main/java/com/yahoo/elide/graphql/QueryRunner.java

```diff
@@ -351,6 +353,36 @@ private ElideResponse executeGraphQLRequest(String baseUrlEndPoint, ObjectMapper
         }
     }
 
+    /**
+     * Generate more user friendly error messages.
+     *
+     * @param errors the errors to map
+     * @return the mapped errors
+     */
+    private List<GraphQLError> mapErrors(List<GraphQLError> errors) {
+        List<GraphQLError> result = new ArrayList<>(errors.size());
+        for (GraphQLError error : errors) {
+            if (error instanceof ValidationError validationError
+                    && ValidationErrorType.WrongType.equals(validationError.getValidationErrorType())) {
+                if (validationError.getDescription().contains("ElideRelationshipOp")) {
+                    String queryPath = String.join(" ", validationError.getQueryPath());
+                    result.add(ValidationError.newValidationError()
+                            .description(
+                                    "Invalid operation: " + queryPath + " is read only.")
```

Comment:
Under what circumstances does this error get generated?  How do we know it is an attempt to write something that is read only?

Reply:
This gets generated in the event an invalid value is used that couldn't map to the GraphQL enum.

I was trying to get the pre-existing test in `NoCacheAggregationDataStoreIntegrationTest` to pass but I guess I should change the expectation instead.